styles/_custom.css

```diff
@@ -1,5 +1,4 @@
 body {
```

Comment:
looks like you have some stuff from the previous commit here riet? Might be worth running `git rebase staging` after first one is merged and pulled